mocks/src/main/kotlin/no/nav/k9/wiremock/WireMocks.kt

```diff
@@ -3,13 +3,14 @@ package no.nav.k9.wiremock
 import com.github.tomakehurst.wiremock.core.Options
 import no.nav.helse.dusseldorf.testsupport.wiremock.WireMockBuilder
 
-internal fun initWireMock(
-        port: Int
+fun initWireMock(
+        port: Int,
+        rootDirectory: String = "../mocks/src/main/resources"
 ) = WireMockBuilder()
         .withPort(port)
         .withAzureSupport()
```

Comment:
Tror denne `.withAzureSupport` burde flyttes ut herfra.
På test-scope i `app` må den legges på, `initWireMock(....).withAzureSupport()` -  men  for  mocks-serveren så bør man ikke ha med azureSupport, men heller starte `azure-mock` docker-imaget.

Det  imaget støtter også Saksbehandler-innlogging som  ikke denne .withAzureSupport() gjør